plugins/inputs/s7comm/README.md

```diff
@@ -0,0 +1,77 @@
+# Siemens S7 Input Plugin
+
+This plugin reads metrics from Siemens PLCs via the S7 protocol.
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml @sample.conf
+# Plugin for retrieving data from Siemens PLCs via the S7 protocol (RFC1006)
+[[inputs.s7comm]]
+  ## Parameters to contact the PLC (mandatory)
+  server = "127.0.0.1:102"
```

Comment:
Is a the `IP + port` syntax required or obvious to users?

Reply:
I will make port 102 the default if no port is given... Good catch!